packages/surveys/src/types.ts

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2020 Environmental Systems Research Institute, Inc.
+ * Apache-2.0 */
+
+import { IRequestOptions } from "@esri/arcgis-rest-request";
+import { IItem } from "@esri/arcgis-rest-types";
+
+type IResultsAvailability = "after" | "always";
+
+export interface IFormItemProperties {
+  [propName: string]: any;
+  previewUrl?: string;
+  parentId?: string;
+  settings?: {
+    resultsAvailability: IResultsAvailability;
+  };
+}
+
+export interface IFormItem extends IItem {
+  id: string;
+  type: "Form";
+  properties?: IFormItemProperties;
+}
+
+type IStakeholderItemTypeKeywords =
```

Comment:
Not sure there's a lot of value in this type since any `string` will do. Could just define `typeKeywords: string[]` on `IStakeholderItem`.

Reply:
Yea, i was hoping to find a typescript-y way to enforce that `IStakeholderItem.typeKeywords` must include those listed, in addition to any others